src/applications/disability-benefits/all-claims/pages/toxicExposure/toxicExposureChoicePage.jsx

```diff
@@ -0,0 +1,358 @@
+import React, { useRef, useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import FormNavButtons from 'platform/forms-system/src/js/components/FormNavButtons';
+import {
+  VaCheckboxGroup,
+  VaModal,
+  VaAlert,
+} from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+import { scrollAndFocus, scrollToFirstError } from 'platform/utilities/ui';
+import cloneDeep from 'platform/utilities/data/cloneDeep';
+import {
+  conditionsDescription,
+  conditionsPageTitle,
+  conditionsQuestion,
+  makeTEConditionsUISchema,
+} from '../../content/toxicExposure';
+import { formTitle } from '../../utils';
+import {
+  deletedToxicExposureAlertConfirmationContent,
+  deleteToxicExposureModalContent,
+  deleteToxicExposureModalDescription,
+  deleteToxicExposureModalTitle,
+} from '../../content/toxicExposureChoiceContent';
+
+export const toxicExposureChoicePageTitle =
+  'Option to claim conditions related to toxic exposure';
+
+/**
+ * List of all toxic exposure related keys that need to be deleted
+ * when user opts out of toxic exposure claims.
+ * @constant {string[]}
+ */
+export const toxicExposureKeys = [
+  'conditions',
+  'gulfWar1990',
+  'gulfWar1990Details',
+  'gulfWar2001',
+  'gulfWar2001Details',
+  'herbicide',
+  'herbicideDetails',
+  'herbicideOtherLocations',
+  'additionalExposures',
+  'additionalExposuresDetails',
+  'specifyOtherExposures',
+];
+
+export const sectionsOfToxicExposure = [
+  'Toxic exposure conditions',
+  'Gulf War service locations and dates (1990 and 2001)',
+  'Agent Orange exposure locations and dates',
+  'Other toxic exposure details and dates',
+];
+
+const ToxicExposureChoicePage = ({
+  goBack,
+  goForward,
+  data,
+  setFormData,
+  contentBeforeButtons,
+  contentAfterButtons,
+  onReviewPage,
+  updatePage,
+}) => {
+  const [
+    showDeleteToxicExposureModal,
+    setShowDeleteToxicExposureModal,
+  ] = useState(false);
+  const [
+    showDeletedToxicExposureConfirmation,
+    setShowDeletedToxicExposureConfirmation,
+  ] = useState(false);
+
+  const deletedToxicExposureConfirmationRef = useRef(null);
+
+  useEffect(
+    () => {
+      if (
+        showDeletedToxicExposureConfirmation &&
+        deletedToxicExposureConfirmationRef.current
+      ) {
+        deletedToxicExposureConfirmationRef.current.focus();
+      }
+    },
+    [showDeletedToxicExposureConfirmation],
+  );
+
+  const modalRef = useRef(null);
+  useEffect(
+    () => {
+      if (showDeleteToxicExposureModal && modalRef.current) {
+        const modalHeading = document.querySelector('h4');
```

Comment:
I assume this is how we did it elsewhere but do we maybe need a more specific query selector if we're querying directly on document? (Since there could be other h4s now or in the future)